pkg/templates/util.go

```diff
@@ -40,18 +41,15 @@ func KubernetesToYAML(data []interface{}) (string, error) {
 			err         error
 		)
 
-		if str, ok := item.(string); ok {
-			encodedItem = []byte(strings.TrimSpace(str))
-		} else {
-			encodedItem, err = yaml.Marshal(item)
-		}
-
+		encodedItem, err = yaml.Marshal(item)
```

Comment:
I don't understand what was the old code doing? Can you explain to me what's the difference here?

Reply:
because input was `[]inetrface`, under the hook it can be anything including `string`, which was type-asserted. Since string doesn't need to be marshalred — no marshaling actually happened. So, basically leftover from old-times when we were running on static yaml files.

Reply:
Okay, that makes sense. Thanks for explaining!